ci/run.sh

```diff
@@ -0,0 +1,64 @@
+# Test our implementation
+case $1 in
+    thumb*)
+        curl -sf "https://raw.githubusercontent.com/japaric/rust-everywhere/master/install.sh" | \
+            bash -s -- --at /usr/bin --from japaric/xargo --tag v0.1.10
+        xargo build --target $1
+        xargo build --target $1 --release
+        ;;
+    # QEMU crashes even when executing the simplest cross compiled C program:
+    # `int main() { return 0; }`
+    powerpc64le-unknown-linux-gnu)
+        cargo test --target $1 --no-run
+        cargo test --target $1 --no-run --release
+        ;;
+    *)
+        cargo test --target $1
+        cargo test --target $1 --release
+        ;;
+esac
+
+# Verify that we haven't drop any intrinsic/symbol
+case $1 in
+    thumb*)
+        xargo build --features c --target $1 --bin intrinsics
+        ;;
+    *)
+        cargo build --features c --target $1 --bin intrinsics
```

Comment:
`cargo test --features c` would end testing the same thing I think. We'll just be testing less intrinsics on some targets. Also, this would require sprinkling more cfg over the test modules.